content/en/blog/_posts/2016-09-00-Kubernetes-1-4-Making-It-Easy-To-Run-On-Kuberentes-Anywhere.md

```diff
@@ -2,7 +2,7 @@
 title: " Kubernetes 1.4: Making it easy to run on Kubernetes anywhere "
 date: 2016-09-26
 slug: kubernetes-1.4-making-it-easy-to-run-on-kuberentes-anywhere
-url: /blog/2016/09/Kubernetes-1-4-Making-It-Easy-To-Run-On-Kuberentes-Anywhere
+url: /blog/2016/09/Kubernetes-1-4-Making-It-Easy-To-Run-On-Kubernetes-Anywhere
```

Comment:
Don't do this.

- the file name is there, so the url follows its wrong spelling;
- we don't fix blogs more than 1 year old.